new/LIT6754PPQwertmat.xml

```diff
@@ -0,0 +1,71 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6754PPQwertmat" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Protective prayer against rheumatism (?) (<foreign xml:lang="gez">qʷǝrṭmāt</foreign>),
```

Comment:
One ǝ is missing in qʷǝrṭmāt (three consecutive consonants are not allowed)

Reply:
True. Strelcyn has qʷǝrṭǝmāt, so let's follow him?

Reply:
Let's follow Strelcyn!